packages/fast-usdc/test/fast-usdc.contract.test.ts

```diff
@@ -510,115 +542,460 @@ test('LP repay', async t => {
+  const { calculateAdvance, calculateSplit } = makeFeeTools(feeConfig);
+  const advance = calculateAdvance(usdc.make(sent1.tx.amount));
+  t.log('advancer sent to PoolAccount', advance);
   t.deepEqual(inspectBankBridge().at(-1), {
-    amount: String(expectedAdvance.value),
-    denom: 'ibc/usdconagoric',
+    amount: String(advance.value),
+    denom: uusdcOnAgoric,
     recipient: 'agoric1fakeLCAAddress',
     type: 'VBANK_GIVE',
   });
 
-  // ibc transfer sent over localChain bridge
-  const localBridgeMsg = inspectLocalBridge().at(-1);
-  const ibcTransferMsg = localBridgeMsg.messages[0];
-  t.is(ibcTransferMsg['@type'], '/ibc.applications.transfer.v1.MsgTransfer');
-
-  const expectedReceiver = addressTools.getQueryParams(
-    evidence.aux.recipientAddress,
-  ).EUD;
-  t.is(ibcTransferMsg.receiver, expectedReceiver, 'sent to correct address');
-  t.deepEqual(ibcTransferMsg.token, {
-    amount: String(expectedAdvance.value),
-    denom: 'ibc/usdconagoric',
-  });
+  cust1.checkSent(t, since(bridgePos));
 
-  // TODO #10445 expect PFM memo
-  t.is(ibcTransferMsg.memo, '', 'TODO expecting PFM memo');
+  const emptyMetrics = {
+    encumberedBalance: usdc.makeEmpty(),
+    shareWorth: {
+      numerator: usdc.make(1n),
+      denominator: { value: 1n },
+    },
+    totalBorrows: usdc.makeEmpty(),
+    totalContractFees: usdc.makeEmpty(),
+    totalPoolFees: usdc.makeEmpty(),
+    totalRepays: usdc.makeEmpty(),
+  };
+  const par250 = {
+    numerator: usdc.make(250_000_001n),
+    denominator: { value: 250_000_001n },
+  };
 
-  // TODO #10445 expect routing through noble, not osmosis
-  t.is(
-    ibcTransferMsg.sourceChannel,
-    fetchedChainInfo.agoric.connections['osmosis-1'].transferChannel.channelId,
-    'TODO expecting routing through Noble',
+  t.like(
+    await E(metricsSub)
+      .getUpdateSince()
+      .then(r => r.value),
+    {
+      ...emptyMetrics,
+      encumberedBalance: advance,
+      shareWorth: par250,
+      totalBorrows: advance,
+    },
+    'metrics while advancing',
   );
 
+  await mint(sent1);
+
+  // C8 - "Contract MUST be able to initialize settlement process when Noble mints USDC."
+  // The metrics are a useful proxy, but the contract could lie.
+  // The real test of whether the contract turns minted funds into liquidity is
+  // the ability to advance the funds (in later tests).
+  const split = calculateSplit(usdc.make(sent1.tx.amount));
+  t.like(
+    await E(metricsSub)
+      .getUpdateSince()
+      .then(r => r.value),
+    {
+      ...emptyMetrics,
+      shareWorth: {
+        ...par250,
+        numerator: add(par250.numerator, split.PoolFee),
+      },
+      totalBorrows: { value: 105839999n },
+      totalContractFees: { value: 432000n },
+      totalPoolFees: { value: 1728001n },
+      totalRepays: { value: 105839999n },
+    },
+    'metrics after advancing',
+  );
+});
+
+test.todo(
+  'PERF: Target: settlement completes in a few minutes (after USDC is minted)',
```

Comment:
I don't think this will be the right place for this, since much of the latency is dependent on blocks and bridges, so maybe we should omit this todo. Would we do this in `multichain-testing`? a testnet?

Reply:
> I don't think this will be the right place for this,

I agree, but this is where I was taking notes on product requirements. I'll take this one out, but note that there are also TODOs here that should be satisfied by exo tests. When the TODOs get done, we can prune these notes.

> since much of the latency is dependent on blocks and bridges, so maybe we should omit this todo. Would we do this in multichain-testing? a testnet?

We have plans for bootstrap tests to measure computrons as well as multichain-testing and testnets.